usaspending_api/etl/data_diff_tool.py

```diff
@@ -0,0 +1,42 @@
+from gresearch.spark.diff import *
+
```

Comment:
RE: the python module (aka the .py file name). I don't think this needs to stand alone in its own python module (file). Can you move this to the `usaspending_api/common/etl/spark.py` utilities module, as a utility that could be used in a data processing workflow?